Chausie/TabPageViewController.swift

```diff
@@ -111,10 +115,36 @@ open class TabPageViewController: UIViewController, PageViewControllerDelegate {
         fatalError("`init(coder:)` has not been implemented")
     }
 
+    open override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+
+        let tabViewWidth = view.bounds.size.width
+            - inset(for: tabViewLayout.leading).left - tabViewLayout.leading.distance
+            - inset(for: tabViewLayout.trailing).right - tabViewLayout.trailing.distance
+        let tabViewSize = CGSize(width: tabViewWidth, height: tabViewLayout.height)
+        if tabViewSize != tabView.bounds.size {
```

Comment:
You need to call `tabView.highlightCells (withRatio:)` after tab view frame calculation is finished (e.g. after device rotation), so execute the process in `viewDidLayoutSubviews()`